documentation/contributing/pullrequests.markdown

```diff
@@ -56,16 +54,18 @@ The ``*-devel`` branches must not be force-pushed.
 No one is allowed to directly commit to the ``*-devel`` branches of the repositories. Instead they should create feature branches and add pull-requests.
 The only commits that might be pushed directly are cherry-picks from older branches (see below) and administrative changes (e.g. CHANGELOGS, tags).
 
-### 1+1 == Merge
+### Two "Approves" == Merge
 
-If you see a trustworthy "+1" note in a request or the requestor is a fellow maintainer and all feedback has been addressed by the requestor,
-merge the request after your own review. Otherwise add a "+1" note after you are satisfied after the review.
+If you see a trustworthy approval review in a request or the requestor is a fellow maintainer and all feedback has been addressed by the requestor,
+merge the request after your own review. Otherwise submit an "Approval" review after you are satisfied after the review.
 
 ### Squash-Merge If Possible
 
 Pull-requests with a single commit should *always* be squash-merged (github supports this via a drop-down list on the "merge" button).
 If a request contains "fixup commits" it should also be squash-merged. If you are unsure this is to the requestor's liking, ask him.
 
+Note: Github recently added a [Rebase and merge](https://github.com/blog/2243-rebase-and-merge-pull-requests) option for approving pull requests - this is likely the recommended method going forward, but this has not been finalized.
+
```

Comment:
I'm not sure what you mean by *approving* requests... it is an alternative to the paragraph above where you recommend always using squash-merge. But I don't think everyone has fully tried/discussed the new method (I think it should be *the* recommended method, soon). 

In terms of merging a request, yes, rebase and merge is an option.

> transient information

I've very active in updating the moveit website, as evidenced by my recent commit history. I think transient info is way better than no info, and in the future it reminds someone to update the snippet with newer information as it develops.

You are welcome to reword the two paragraphs into one cohesive policy, I just wasn't 100% we were ready for that.